doc/src/build/wallet.md

```diff
@@ -27,29 +27,51 @@ in [Genesis customization](#customize-genesis).
 1. Optionally, set `RUST_LOG=debug` for verbose logging.
 1. Initiate `genesis`:
    ```shell
-   sui genesis
+   $ sui genesis
    ```
 
+All of this is contained in configuration and keystore files, as well
+as an `authorities_db` database file.
+TODO: Why do I no longer see a `client_db` file created, only `authorities_db` now?
+
 The network configuration is stored in `network.conf` and can be used
 subsequently to start the network. The `wallet.conf` and `wallet.key`
 are also created to be used by the Sui wallet to manage the newly
-created accounts. By default, these files will be stored in the
-`~/.sui/sui_config` directory, but you can override this location by
-providing an alternative path:
+created accounts.
+
+By default, these files are placed in your home directory at
```

Comment:
```suggestion
By default, these files are placed in the
```
It's obvious from the path that it's home directory plus "directory" was used twice.

Reply:
I removed the second "directory" here. Thanks for the catch.